packages/vue/src/components/__tests__/setup-totp.spec.ts

```diff
@@ -119,8 +119,8 @@ describe('SetupTotp', () => {
     render(SetupTotp, { global: { components } });
 
     // wait for qrcode to render
-    await screen.findByAltText('qr code');
-
+    waitFor(() => screen.findByAltText('qr code'));
+    // waitFor(() => expect(consoleWarnSpy).toHaveBeenCalled());
```

Comment:
should this be commented out?

Reply:
Oh that shouldn't even be there, took it out